tests/test_config_file_deep_merge.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import annotations as _annotations
+
+import json
+import sys
+from pathlib import Path
+from typing import Optional
+
+import pytest
+from pydantic import AnyHttpUrl, Field
+
+try:
+    import yaml  # type: ignore
+except Exception:
+    yaml = None
+
+try:
+    import tomli  # type: ignore
+except Exception:
+    tomli = None
+
+from pydantic_settings import (
+    BaseSettings,
+    JsonConfigSettingsSource,
+    PydanticBaseSettingsSource,
+    SettingsConfigDict,
+    TomlConfigSettingsSource,
+    YamlConfigSettingsSource,
+)
+
+
+def test_init_kwargs_override_env_for_alias_with_populate_by_name(monkeypatch):
```

Comment:
Please use the `env` fixture that we have in our setup.
like 
https://github.com/pydantic/pydantic-settings/blob/3e664300f38a4ad1e1ff4b431147f18d63930028/tests/test_settings.py#L156